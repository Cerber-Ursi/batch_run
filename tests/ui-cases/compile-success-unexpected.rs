fn main() {
    println!("It works, unexpectedly!");
}
