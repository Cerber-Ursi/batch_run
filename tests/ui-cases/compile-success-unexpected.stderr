error: ERROR
 --> tests/ui-cases/compile-fail-ok.rs:1:1
  |
1 | compile_error!("ERROR");
  | ^^^^^^^^^^^^^^^^^^^^^^^^
